includes/Admin/Settings_Page.php

```diff
@@ -169,12 +169,16 @@ private static function output_import_products_modal_template() {
 				<div class="wc-backbone-modal-content">
 					<section class="wc-backbone-modal-main" role="main">
 						<header class="wc-backbone-modal-header">
-							<h1><?php esc_html_e( 'Import Products From Square', 'woocommerce-square' ); ?></h1>
+							<h1><?php esc_html_e( 'Import New Products From Square', 'woocommerce-square' ); ?></h1>
 							<button class="modal-close modal-close-link dashicons dashicons-no-alt">
 								<span class="screen-reader-text"><?php esc_html_e( 'Close modal window', 'woocommerce-square' ); ?></span>
 							</button>
 						</header>
 						<article>
+							<p style="background: #f0f6fc; border-left: 4px solid #2271b1; padding: 10px 12px; margin-bottom: 16px;">
+								<strong><?php esc_html_e( 'This is for importing new products.', 'woocommerce-square' ); ?></strong>
+								<?php esc_html_e( 'To update existing synced products, use "Sync Now" on the Update tab instead.', 'woocommerce-square' ); ?>
+							</p>
```

Comment:
Import products option also provide a way to update existing products while doing import, So, this is not accurate.

<img width="1090" height="627" alt="Image" src="https://github.com/user-attachments/assets/1153d54f-3b52-4e41-ac8f-75b4b9d2791c" />